### PR TITLE
Add Min browser

### PR DESCRIPTION
**Pull request by jfsiii**

### CODE REVIEW

File: _data/apps.yml [hunk 1]
```diff
@@ -1081,3 +1081,13 @@
...
+-
+  name: "Min"
+  description: "A smarter web browser"
+  website: "https://minbrowser.github.io/min/"
+  repository: "https://github.com/minbrowser/min"
+  keywords:
+    - "browser"
+    - "privacy"
+  license: "Apache 2.0"
+  icon: "icon256.png"
```
Comment (kevinsawicki) [inline _data/apps.yml]: Looks like the icon didn't get added to this pull request 📷

Comment (jlord) [inline _data/apps.yml]: I don't see an icon included with this PR? Could you add an icon and give it a name to match your app? Thanks!

Comment (jfsiii) [inline _data/apps.yml]: @kevinsawicki How do I do that? https://github.com/electron/electron.atom.io/blob/gh-pages/CONTRIBUTING.md#adding-an-app-or-project-to-the-site doesn't mention it. Should I add it to https://github.com/electron/electron.atom.io/tree/gh-pages/images/apps ?

Comment (kevinsawicki) [inline _data/apps.yml]: > Should I add it to https://github.com/electron/electron.atom.io/tree/gh-pages/images/apps ?

Yup, thanks, will update the docs.

Comment (jfsiii) [inline _data/apps.yml]: Great, thanks. I've uploaded the icon and updated the path.

**Comment by kevinsawicki** [comment]
Awesome, thanks, looks really cool 👍

**Pull request merged**